Modules/User/changelog.yml

```diff
@@ -1,5 +1,8 @@
 url: https://github.com/AsgardCms/Platform
 versions:
+ "@unreleased":
+         changed:
+         - Adding a test the user token is correctly generated
```

Comment:
Indentation is wrong here.
Not sure why you added this aswell as this was done before.